Controller/FormController.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
```

Comment:
please use the header defined in https://github.com/desarrolla2/FormBundle/blob/master/.formatter.yml